client/src/connection/rest/common.ts

```diff
@@ -0,0 +1,143 @@
+import * as typer from "media-typer";
```

Comment:
Can we import what we need from `media-typer` instead of all of the things?

Reply:
Yes, I did that because the function names in media-typer we very generic and can easily collide with others. I will import the functions as a clear name though.